lib/nuca_backend_web/controllers/hotspot_controller.ex

```diff
@@ -37,8 +37,8 @@ defmodule NucaBackendWeb.HotspotController do
     render(conn, "edit.html", hotspot: hotspot, changeset: changeset)
   end
 
-  def update(conn, %{"id" => id, "hotspot" => hotspot_params}) do
-    hotspot = Hotspots.get_hotspot!(id)
+  def update(conn, hotspot_params) do
+    hotspot = Hotspots.get_hotspot!(hotspot_params["id"])
```

Comment:
hm, this is not very "RESTfull"
an update request should be: `PATCH /hotspots/:id` and the payload is a JSON in the body of the request
So this code was good actually, and we need to change how we send the request from the front-end.

Reply:
disregard, I read it wrong